var/spack/repos/builtin/packages/cmaq/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cmaq(Package):
+    """Code base for the U.S. EPA’s Community Multiscale Air Quality Model
```

Comment:
```suggestion
    """Code base for the U.S. EPA's Community Multiscale Air Quality Model
```
Python 2 doesn't play nicely with unicode, hence the failing unit tests.